doc/user/content/manage/dbt/development-workflows.md

```diff
@@ -312,21 +312,30 @@ These environments are later swapped transparently.
 
 <br>
 
-1. In `dbt_project.yml`, use the `deployment` variable to specify the cluster(s)
-   and schema(s) that contain the changes you want to deploy.
+1. The `deploy_get_objects` macro automatically identifies the clusters and schemas in your dbt project. If you need to exclude specific clusters or schemas from the deployment, you can specify them in the `dbt_project.yml` file:
 
     ```yaml
     vars:
       deployment:
         default:
-          clusters:
+          exclude_clusters:
             # To specify multiple clusters, use [<cluster1_name>, <cluster2_name>].
-            - <cluster_name>
-          schemas:
-            # to specify multiple schemas, use [<schema1_name>, <schema2_name>].
-            - <schema_name>
+            - <cluster_to_exclude>
+          exclude_schemas:
+            # To specify multiple schemas, use [<schema1_name>, <schema2_name>].
+            - <schema_to_exclude>
     ```
 
+   If you don't need to exclude any clusters or schemas, you can omit this configuration entirely.
+
+1. To quickly view the clusters and schemas that will be included in the deployment, you can use the [`deploy_get_objects`](https://github.com/MaterializeInc/materialize/blob/main/misc/dbt-materialize/dbt/include/materialize/macros/deploy/deploy_get_objects.sql) macro with the `debug` argument:
+
+  ```bash
+  dbt run-operation deploy_get_objects --args '{debug: true}'
+  ```
+
+  This will output the list of clusters and schemas that will be included in the deployment, as well as any that have been excluded based on your configuration.
```

Comment:
Tiny alignment issue. I believe these lines need to be shifted by 1 to the right as the current alignment causes a break in numbering.
<img width="529" alt="Screenshot 2024-09-11 at 2 07 26 PM" src="https://github.com/user-attachments/assets/56b67572-a975-4131-9ee3-087f21ba502c">

Reply:
Ah yes, good catch! Thanks @kay-kim